### PR TITLE
Update amp-usernotication style

### LINKED ISSUE #739: use AMP Start design for notification
image

there is an AMP Start one that @camelburrito implemented but is not in use on AMP Start. that one matches the site design so it looks more intentional.

### PR DESCRIPTION
**Pull request by sebastianbenz**
* uses ampstart style
* fixes #739

### CODE REVIEW

**Comment by sebastianbenz** [comment]
screen shot 2017-05-02 at 1 26 16 pm

File: src/30_Advanced/amp-user-notification_with_Server_Endpoint.html [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
   <style amp-custom>
-    footer {
-      margin-bottom: 50px;
-    }
-    amp-user-notification {
-      padding: 8px;
-      background: beige;
+    /* Center the notification content */
+    amp-user-notification > div {
+      padding: 1rem;
```
Comment (kul3r4) [inline src/30_Advanced/amp-user-notification_with_Server_Endpoint.html]: could we use p1 instead of padding: 1rem;?

Comment (sebastianbenz) [inline src/30_Advanced/amp-user-notification_with_Server_Endpoint.html]: thought about the same - in this case we have all relevant styling in once place. We need some clear rules on when to use basscss and when not.

Comment (kul3r4) [inline src/30_Advanced/amp-user-notification_with_Server_Endpoint.html]: ok! we can follow this convention when we create styling classes.

**Pull request merged**